spec/services/content_validator_spec.rb

```diff
@@ -5,7 +5,7 @@
 RSpec.describe ContentValidator do
   describe 'title validation' do
     it 'raises issue if the title is not set' do
-      document = build(:document, title: nil)
+      document = build(:document, title: nil, base_path: nil)
```

Comment:
Why are we unsetting the base_path here? (The test description only mentions the title.)